src/core/settings.h

```diff
@@ -40,6 +40,7 @@ typedef struct QUIC_SETTINGS_INTERNAL {
             uint64_t PacingEnabled                          : 1;
             uint64_t MigrationEnabled                       : 1;
             uint64_t DatagramReceiveEnabled                 : 1;
+            uint64_t ThisReliableResetFrameEnabled          : 1;
```

Comment:
RESERVED to be 22

Reply:
Sorry, I don't understand. Are you saying IsSet->RESERVED should be 22 instead of 23?

Reply:
As you add new bits, you effectively take away reserved bits, so the count of reserved should reduce.

Reply:
That makes a lot of sense. I was wondering what RESERVED meant lol. Will update.